src/recursion.js

```diff
@@ -3,17 +3,130 @@
 const nFibonacci = (n) => {
   // fibonacci sequence: 1 2 3 5 8 13 ...
   // return the nth number in the sequence
+  if (n < 0) {
```

Comment:
I think you can simplify all of this logic to one single line. `if (n <= 2) return n`